Content.Server/Weapon/Ranged/Ammunition/Components/AmmoComponent.cs

```diff
@@ -114,6 +114,7 @@ public enum BallisticCaliber
         Pistol,
         A35, // Placeholder?
         LRifle,
+        HRifle,
```

Comment:
this will conflict with w/e gun refactor silver is doing but thats laright

Reply:
Silver is going to pull their fixes to my branch I believe.